apps/web/pages/api/user/avatar.ts

```diff
@@ -62,6 +64,9 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   const img = identity?.avatar;
   // If image isn't set or links to this route itself, use default avatar
   if (!img) {
+    if (identity?.org) {
+      res.setHeader("x-cal-org", identity.org);
```

Comment:
It's helpful for debugging. One can just send the request and know if the domain has been labelled as an org subdomain or not.